docs/LOCALIZATION.md

```diff
@@ -0,0 +1,248 @@
+   -  Copy `en-US.json` as the base template.
+
+## Updating strings of an existing language
+
+There are two types of supported languages:
+
+-  Maintained by Microsoft
+-  Contributed by the community
+
+For strings that are maintained by Microsoft, please file a bug to us. It will take us time as we need perform additional checks to maintain the quality.
+
+For strings that are contributed by the community, please submit a pull request to us.
+
+## Adding new strings for new UI
+
+To add new strings to be used in the UI, please update `/packages/component/src/localization/en-US.json`. Our localization team will pick up the new strings and translate them to all other officially supported languages.
+
+## Overriding localization strings
+
+Developers can override localization strings in one of two ways. An `overrideLocalizedStrings` prop will be passed and it can either be a plain object or a function.
+
+```js
+window.WebChat.renderWebChat(
+   {
+      locale: 'en-US',
+
+      overrideLocalizedStrings: (strings, language) => ({
+         ...strings,
+         TEXT_INPUT_PLACEHOLDER: 'What is on your mind?'
+      }),
+
+      overrideLocalizedStrings: {
+         TEXT_INPUT_PLACEHOLDER: 'What is on your mind?'
+      }
+   },
+   document.getElementById('webchat')
+);
+```
+
+## Design
+
+### User stories
+
+The localization feature is backed by these user stories:
+
+-  Translated by Microsoft localization team with automated pipeline
+-  Overrideable by developers
+   -  Change the "Type your message" placeholder text
+
+### Translated by Microsoft localization team
+
+We support 44 languages, including right-to-left languages, translated by Microsoft localization team. Additional languages by the community are always welcomed.
+
+| Language Code | Language              | Name             |
+| ------------- | --------------------- | ---------------- |
+| `ar-SA`       | Arabic (Saudi Arabia) | اَلْعَرَبِيَّةُ  |
+| `eu-ES`       | Basque                | euskara          |
+| `bg-BG`       | Bulgarian             | Български        |
+| `ca-ES`       | Catalan               | català           |
+| `zh-Hans`     | Chinese (Simplified)  | 中文(简体)       |
+| `zh-Hant`     | Chinese (Traditional) | 中文(繁體)       |
+| `hr-HR`       | Croatian              | hrvatski         |
+| `cs-CZ`       | Czech                 | čeština          |
+| `da-DK`       | Danish                | dansk            |
+| `nl-NL`       | Dutch                 | Nederlands       |
+| `en-US`       | English               | English          |
+| `et-EE`       | Estonian              | eesti            |
+| `fi-FI`       | Finnish               | suomi            |
+| `fr-FR`       | French                | français         |
+| `gl-ES`       | Galician              | galego           |
+| `de-DE`       | German                | Deutsch          |
+| `el-GR`       | Greek                 | Ελληνικά         |
+| `he-IL`       | Hebrew                | עברית            |
```

Comment:
Clean up table spacing

Reply:
Actually it's clean, but some letters are not monospace, thus, it looks weird.